XenModel/Messages.resx

```diff
@@ -10753,7 +10750,16 @@ This will cancel the rolling pool upgrade process and may leave your system in a
   <data name="ROLLING_UPGRADE_ERROR_POOL_ONE" xml:space="preserve">
     <value>The following error occurred while rolling pool upgrade was in progress:</value>
   </data>
+  <data name="ROLLING_UPGRADE_EXTRAS_PAGE_TEXT" xml:space="preserve">
+    <value>Extras</value>
+  </data>
+  <data name="ROLLING_UPGRADE_EXTRAS_PAGE_TITLE" xml:space="preserve">
+    <value>Choose whether to also apply updates or suplemental packs</value>

```

Comment:
`supplemental`
We could make this a bit shorter since it's a title, i.e. `Optionally, apply updates or supplemental packs` - we have this elsewhere too